scripts/zones/Mount_Zhayolm/mobs/Cerberus.lua

```diff
@@ -37,5 +37,5 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    mob:setRespawnTime((math.random(0,24)*3600)+172800); -- 48-72 hours proper 1 hour windows
+    mob:setRespawnTime(math.random(172800,259200)); -- 48-72 hours proper 1 hour windows
```

Comment:
This was technically correct as it was though.

(and nearly every other one isn't - its just hardly anybody cares about those exact windows so nobody ever pull requested to change them)